### ISSUE TITLE
Passing config_file arg at launch results on failing to parse the argument

### ISSUE THREAD

**Issue by TSC21** [realsenseai/realsense-ros#1913]
Trying `ros2 launch realsense2_camera rs_launch.py config_file:="'<abs_path_to_file>/d435i.yaml'"` results on:

```sh
[WARNING] [launch_ros.actions.node]: Parameter file path is not a file: '<abs_path_to_file>/d435i.yaml'
```

I believe the current logic in `rs_launch.py` using the substitutions with `LaunchConfiguration` is faulty. I tried some different approach but no luck so far. I believe this needs a fix.

**Comment by TSC21** [comment]
Found the solution in https://github.com/IntelRealSense/librealsense/issues/8863#issuecomment-823026235. Issuing a PR for it.

**Comment by MartyG-RealSense** [comment]
Thanks very much @TSC21 - if you create a PR, please post a link to it here when it is submitted.  Thanks again!

**Comment by TSC21** [comment]
> Thanks very much @TSC21 - if you create a PR, please post a link to it here when it is submitted. Thanks again!

@MartyG-RealSense here: https://github.com/IntelRealSense/realsense-ros/pull/1915. Happy to help!

**Comment by MartyG-RealSense** [comment]
Great, thanks @TSC21 - I have added an **Enhancement** tag to this case to highlight that this case https://github.com/IntelRealSense/realsense-ros/issues/1913 should be kept open whilst the associated PR is active.  Thanks again!

**Comment by doronhi** [comment]
Fixed in #1942
